.github/workflows/monitor-r2-stats.yml

```
@@ -23,199 +19,142 @@ jobs:
+      - name: Send Critical Alert on High Usage
+        if: steps.metrics.outputs.warnings > 0
         env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
         run: |
-          cat << EOF > slack-notify.js
-          const { IncomingWebhook } = require('@slack/webhook');
-          const webhook = new IncomingWebhook(process.env.SLACK_WEBHOOK_URL);
+          cat << EOF > critical-alert.js
+          const https = require('https');
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)